scripts/build/watch.js

```diff
@@ -76,8 +78,10 @@ const run = ({ cmd, args }) => {
 };
 
 const startWatchers = () => {
+  console.log('starting watchers');
   for (const config of configs) {
     const watcher = new Gaze(config.files, GAZE_OPTIONS);
+    console.log({ watched: watcher.watched() });
```

Comment:
This feels like too much noise and not very useful. Consider removing this line?

Reply:
Sorry, that was careless on my part.

Reply:
@garethbowen updated